packages/components/src/scroll-to/README.md

```diff
@@ -0,0 +1,15 @@
+ScrollTo
+===
+
+
+
+## Usage
+
+```jsx
+```
+
+### Props
+
+Name | Type | Default | Description
+--- | --- | --- | ---
+`offset` | String | ``'0'`` | The offset from the top of the component
```

Comment:
This file seems incomplete, is it something that gets filled out in later iterations?

Reply:
There was an unmerged PR for that #2873